blocks/library/embed/index.js

```diff
@@ -129,31 +154,38 @@ registerBlockType( 'core/embed', {
 			const { url, caption } = this.props.attributes;
 			const { setAttributes, focus, setFocus } = this.props;
 
+			if ( fetching ) {
+				return (
+					<div className="blocks-embed__loading">
+						<Spinner />
+						<p className="blocks-embed__loading-text">{ wp.i18n.__( 'Embedding...' ) }</p>
```

Comment:
Minor: Ellipsis character `…` is more semantic than three dots for this purpose.